crowbar_framework/app/models/backup.rb

```diff
@@ -114,10 +117,10 @@ def create_archive
   end
 
   def save_archive
-    self.name = file.original_filename.split(".").first
+    self.name = file.original_filename.remove(".tar.gz")
```

Comment:
Thanks.

This makes me wonder: should we have a validation when uploading a tarball that the extension is `.tar.gz`?